workspaces/adventure-pack/goodies/java/src/integer_digits/AP.java

```diff
@@ -0,0 +1,30 @@
+package integer_digits;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+
+public final class AP {
+
+  /**
+   * Get the digits in the given integer. For example, in the given integer 8725, digits are
+   * [8, 7, 2, 5].
+   * @param number Any number of interest.
+   * @return The {@link IterableIntStream stream} of digits in the given integer.
+   */
+  public static IterableIntStream integerDigits(int number) {
```

Comment:
Would you be able to add a unit test for this?

Reply:
Sure. I'll add the unit test.

Reply:
Agreed, it would be nice to have some tests here -- have a look at some of the `Test.java` files available for other Java goodies, like `UnionFind`.

Reply:
Addressed the comment